Bottom: 3f2c9d27893a929b86b3161c6c88850092aee94e
Top:    8a46f0fc056b1350d828e278d9f34a52e1bc2ebf
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-11 13:51:27 +0800

Refresh of autofs-5.0.5-wait-for-master-map-available-at-start.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 48e9806..f8a8ceb 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -49,6 +49,7 @@
 - fix master map bogus keywork match.
 - fix fix map entry duplicate offset detection.
 - probe each nfs version in turn for singleton mounts.
+- wait for master map available at start.
 
 25/07/2012 autofs-5.0.7
 =======================
